content_loading/function_app.py

```diff
@@ -139,6 +149,21 @@ def __create_llm__() -> AzureOpenAI:
         api_version=api_version
     )
 
+def __create_llama_compat_llm() -> MultiModalLLM:
```

Comment:
Should we fix the type to AzureOpenAIMultiModal or pass in the base class and let the implementor decide which multi modal to use?

Reply:
I prefer minimal signatures for extensibility and least exposure. Liskov substitution and all that good stuff.